x-pack/plugins/infra/server/lib/alerting/metric_threshold/lib/metric_query.test.ts

```diff
@@ -65,19 +71,6 @@ describe("The Metric Threshold Alert's getElasticsearchMetricQuery", () => {
       end,
       start: end - 5 * 60 * 1000,
     };
-    const searchBody = getElasticsearchMetricQuery(
-      expressionParams,
-      timefield,
-      undefined,
-      undefined,
-      timerange
-    );
-    test('by rounding timestamps to the nearest timeUnit', () => {
```

Comment:
Not sure if this was removed on purpose. If it's on purpose I think the `describe` that contains it, can also be removed as there is no other test.

Reply:
Whoops... good catch on the `describe`. This part of the function was moved up a level.